android/build.gradle

```diff
@@ -27,6 +27,9 @@ buildscript {
     if (project.hasProperty("filament_dist_dir")) {
         filamentPath = file(project.property("filament_dist_dir")).absolutePath
     }
+    // CMake scripts require a forward-slash path for the FILAMENT_DIST_DIR
+    // variable, so here we convert the native path to a forward-slash path.
+    filamentPath = filamentPath.replace(File.separator, "/")
```

Comment:
It should be replaceAll right?

Reply:
replaceAll will compile a regex, so in that case it would require again extra back slash, how it was in the begining "\\\\".